src/main/resources/org/jenkinsci/plugins/workflow/log/tee/TeeLogStorageFactory/config.jelly

```diff
@@ -0,0 +1,18 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+  <f:section title="${%Tee Log Storage Factory}">
+    <f:block>
+      ${%description}
+    </f:block>
+    <f:dropdownDescriptorSelector
+        field="primary" title="${%Primary}"
+        descriptors="${descriptor.getFilteredDescriptors()}"
+        default="${descriptor.getDefaultLogStorageFactoryDescriptor()}"
+    />
+    <f:dropdownDescriptorSelector
+        field="secondary" title="${%Secondary}"
+        descriptors="${descriptor.getFilteredDescriptors()}"
+        default="${descriptor.getDefaultLogStorageFactoryDescriptor()}"
+    />
```

Comment:
I don't know your exact use case, but maybe `repeatableHeteroProperty` would be a better fit (select multiple implementations and order them)

Reply:
Yeah, this was used in a previous commit, but it seemed simpler to have only one primary and only one secondary at this point. But maybe additional checks are possible.